testlib/helper.go

```diff
@@ -48,20 +50,41 @@ func NewMainHelper() *MainHelper {
 		store.NewLayeredStore(sqlSupplier, nil, clusterInterface),
 	}
 
+	testResourcePath := SetupTestResources()
+
 	return &MainHelper{
 		Settings:         settings,
 		Store:            testStore,
 		SqlSupplier:      sqlSupplier,
 		ClusterInterface: clusterInterface,
+		TestResourcePath: testResourcePath,
 	}
 }
 
 func (h *MainHelper) Main(m *testing.M) {
+	prevDir, err := os.Getwd()
+	if err != nil {
+		panic("Failed to get current working directory: " + err.Error())
+	}
+
+	err = os.Chdir(h.TestResourcePath)
+	if err != nil {
+		panic(fmt.Sprintf("Failed to set current working directory to %s: %s", h.TestResourcePath, err.Error()))
+	}
+
+	defer func() {
+		err := os.Chdir(prevDir)
+		if err != nil {
+			mlog.Error(fmt.Sprintf("Failed to set current working directory to %s: %s", prevDir, err.Error()))
```

Comment:
Propose clarifying that we're trying to revert the wd here. Also, I think it still makes sense to panic, since something is clearly very wrong.
```suggestion
			panic(fmt.Sprintf("Failed to restore current working directory to %s: %s", prevDir, err.Error()))
```